applications/community-health-check.md

```diff
@@ -0,0 +1,281 @@
+
+While the development of the full dashboard is ongoing, we continue to deliver low fidelity dashboards to interested communities. This lets us interact closely with users, helping us learn more about their problems and shape onboarding material for users. Interested communities get direct access to our team of data scientist who happily investigate follow-up questions. 
+
+Early development of this project has been funded by [Aragon](https://forum.aragon.org/t/real-time-community-health-analytics), Polygon DAO, [Aave](https://noturhandle.gitbook.io/rndao/proposals/proposals/grant-proposal-aave), [MetaCartel](https://forum.metacartel.org/t/grant-request-community-health-a-project-by-rndao/2354) and Near.
+
+## Development Roadmap :nut_and_bolt:
+
+
+### Overview
+
+- **Total Estimated Duration:**  10 weeks
+- **Full-Time Equivalent (FTE):**  6
+- **Total Costs:**  15 000 USDC
+
+We are following a grant-matching processes to spread the risk among different communities. 
+
+### Milestone 1 - Defining metrics
+
+- **Estimated duration:** 4 weeks
+- **FTE:**  1
+- **Costs:** 5000 USDC
+
+Research social network metrics that are viable to measure the health of a community on Twitter, and how they integrate with our existing community health framework. This will build on our existing framework, extending it to metrics that have been tested and used in Twitter communities. For example, we will look into approaches to build the network, virality and clustering/fragmentation of very large online communities. 
+
+Note: We have done an evaluation to decide wheter Reddit or Twitter would be a better option. Given other requests we have received, we have decided that building an integration and health metrics for Twitter is a more viable path for us.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | The results will be published open-access (no paywall or copyright). |
+| 0b. | Documentation | No documentation necessary.|
+| 0c. | Testing and Testing Guide | Testing and Testing Guide not necessary. |
+| 0d. | Docker | A docker file is not necessary for using the community health check. |
+| 0e. | Article| We will publish an article (technical document) describing the metrics, the insights (so-what), and limitations. This article will also explain how the Twitter community is build. |
+
+### Milestone 2 — Twitter community health dashboard
+
+- **Estimated duration:** 6 weeks
+- **FTE:**  5
+- **Costs:** 10 000 USDC
+
+This milestone implements the work from the previous milestone by building the dashboard.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | The code for data fetching and storage aisre already available in a public github folder (Unlicensed). We will specify an OSS license. |
```

Comment:
```suggestion
| 0a. | License | The code for data fetching and storage is already available in a public GitHub folder (Unlicensed). We will specify an OSS license. |
```

Could you add a link to the GitHub repo since it's already available?